tests/manage/z_cluster/upgrade/test_check_osd_post_ocs_upgrade.py

```diff
@@ -0,0 +1,32 @@
+import logging
+
+import pytest
+
+from ocs_ci.framework.testlib import (
+    post_ocs_upgrade,
+    post_ocp_upgrade,
+)
+from ocs_ci.ocs.cluster import validate_osd_pdb
+from ocs_ci.ocs.ocp import get_ocs_version
+from ocs_ci.ocs.resources.pod import get_all_pods
+from ocs_ci.ocs import constants
+
+log = logging.getLogger(__name__)
+
+
+@post_ocp_upgrade
+@post_ocs_upgrade
+@pytest.mark.polarion_id("OCS-2524")
+def test_check_osd_pdb_post_upgrade():
+    """
+    Test to verify OSD PDBs
+    1. Post OCP and OCS successful upgrades check for OSD PDBs
+    2. Rook-ceph-drain-canary pods disappeared after upgrade of OCS version to ocs-operator.v4.6.2-233.ci
+
+    """
+    assert not validate_osd_pdb(), "Blocking PDBs present in the cluster"
```

Comment:
In addition to checking whether blocking PDBs are present or not, we should also check for the default PDB values when there is no drain operation post upgrade.

Reply:
The function validate_existence_of_osds_pdb() also prints osds pdbs in the absence of blocking osd pdbs, IMO i dont think again this check would be necessary